improver/lapse_rate.py

```diff
@@ -256,81 +205,148 @@ def __repr__(self):
                         self.max_lapse_rate, self.min_lapse_rate))
         return desc
 
-    def _calc_lapse_rate(self, temperature, orography):
-        """Function to calculate the lapse rate.
-
-        This holds the function to determine the local lapse rate at a point by
-        calculating a least-squares fit to local temperature and altitude data
-        to find the local lapse rate.
+    def _rolling_window(self, A, temp_shape, shape):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5319684)